erpnext/stock/doctype/serial_and_batch_bundle/serial_and_batch_bundle.py

```
@@ -229,7 +228,7 @@ def validate_serial_nos_inventory(self):
 			return
 
 		if self.voucher_type == "Stock Reconciliation":
-			serial_nos = self.get_serial_nos_for_validate()
+			serial_nos, batches = self.get_serial_nos_for_validate()
```

Ruff (0.14.2)

Unpacked variable `batches` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)